src/encoded/static/components/browse/BrowseView.js

```diff
@@ -511,91 +426,181 @@ export default class BrowseView extends React.Component {
-                                </button>
-                            </div>
-                            : null }
-                        <hr/>
-                    </div>
-                </div>
+                <SelectedFilesController href={href}>
+                    <BrowseTableWithSelectedFilesCheckboxes {...this.props} facets={facets} />
+                </SelectedFilesController>
             </div>
         );
     }
 
+}
+
+/**
+ * Fallback view for no results found.
+ * If no 4DN projects available in this query but there are External Items, let user know.
+ * And, show list of suggested actions.
+ */
+const NoResultsView = React.memo(function NoResultsView({ context, href, browseBaseState, countExternalSets }){
+
+    /** Function to reuse the search function but with External Data flag activated. */
+    const browseExternalData = (e)=>{
+        e.preventDefault();
+        e.stopPropagation();
+        navigate.setBrowseBaseStateAndRefresh('all', href, context);
+    };
+
+    // If there are no External Sets found:
+    // - Tell the user there is no data.
+
+    // If there are External Sets that match the filter:
+    // - Tell the user they exist, and how many.
+    // - Instruct the user to click on the button to search for Experiment Sets in External Data.
+    return (
+        <div className="browse-page-container search-page-container container" id="content">
+            <div className="error-page mt-4">
+                <div className="clearfix">
+                    <hr/>
+                    {
+                        countExternalSets > 0 ?
+                            <h4 className="text-400 mb-18 mt-05">Only External Data results were found.</h4>
+                            :
+                            <h3 className="text-400 mb-05 mt-05">No results found.</h3>
+                    }
+                    { browseBaseState !== 'all' && countExternalSets > 0 ?
+                        <div className="mb-10 mt-1">
+                            <button type="button" className="btn btn-primary text-400 inline-block clickable in-stacked-table-button"
+                                onClick={browseExternalData} data-tip="Keep current filters and browse External data">
+                                Browse <span className="text-600">{ countExternalSets }</span> External Data { countExternalSets > 1 ? 'sets ' : 'set ' }
+                            </button>
+                        </div>
+                        : null }
+                    <hr/>
+                </div>
+            </div>
+        </div>
+    );
+});
+
+
+function BrowseTableWithSelectedFilesCheckboxes(props){
+    const {
+
+        // Common high-level props from Redux, or App.js, or App.js > BodyElement:
+        context, href, browseBaseState, schemas, navigate: propNavigate,
+        windowHeight, windowWidth, registerWindowOnScrollHandler,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['registerWindowOnScrollHandler' is missing in props validation (react/prop-types)](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4660599)